README.md

```diff
@@ -11,41 +11,38 @@
 
 # Development Instructions
 
+## _Optional_ Terminal Installs
 - _(Optional)_ Install [**iTerm**](https://www.iterm2.com/) for a better Command Line App in OS X.
-- _(Optional)_ Install [**oh-my-zsh**](https://github.com/robbyrussell/oh-my-zsh) for a prettier command line interface and easier zsh configuration than bash. - `sh -c "$(curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"`
+- _(Optional)_ Install [**oh-my-zsh**](https://github.com/robbyrussell/oh-my-zsh) for a prettier command line interface and easier zsh configuration than bash. 
```

Comment:
I can't live without oh-my-zsh, but I'd advocate for just getting rid of non essential stuff like this.